docs/logging.rst

```diff
@@ -135,16 +135,21 @@ of the Astropy configuration file::
     log_exceptions = False
 
     # Whether to always log messages to a log file
-    log_to_file = True
+    log_to_file = False
```

Comment:
Changing this does not seem necessary. I think this example is to show a non-default setting.
```suggestion
    log_to_file = True
```